src/interactions/interaction.ts

```diff
@@ -44,7 +44,9 @@ module Plottable {
      * @return {Interaction}
      */
     public detachFrom(component: Component) {
-      this._unanchor();
+      if (this._isAnchored) {
```

Comment:
Wouldn't it make more sense if this `if`-block encompassed all of the logic, rather than just `_unanchor()`? Although I don't really expect anything bad to happen.